test/integration/targets/canonical-pep517-self-packaging/runme_test.py

```diff
@@ -209,7 +312,7 @@ def test_dist_rebuilds_with_manpages_premutations(
     # Checking that the expected sdist got created
     # from the previous unpacked sdist...
     assert tmp_path_rebuilt_sdist.exists()
-    assert contains_man1_pages(tmp_path_rebuilt_sdist)
+    # assert contains_man1_pages(tmp_path_rebuilt_sdist)  # FIXME: See #80255
```

Comment:
Could you update this comment to reflect the changes you'd like to make in the future?

Reply:
I'll add an extra comment on the line above since I prefer having a "fixme" on this line but don't want to have long lines.

Reply:
```suggestion
    # NOTE: The following assertion is disabled due to the fact that, when
    # NOTE: building an sdist from the original source checkout, the build
    # NOTE: backend replaces itself with pure setuptools in the resulting
    # NOTE: sdist, and the following rebuilds from that sdist are no longer
    # NOTE: able to process the custom config settings that are implemented in
    # NOTE: the in-tree build backend. It is expected that said
    # NOTE: `pyproject.toml` mutation change will be reverted once all of the
    # NOTE: supported `ansible-core` versions ship wheels, meaning that the
    # NOTE: end-users won't be building the distribution from sdist on install.
    # NOTE: Another case, when it can be reverted is declaring pip below v20
    # NOTE: unsupported — it is the first version to support in-tree build
    # NOTE: backends natively.
    # assert contains_man1_pages(tmp_path_rebuilt_sdist)  # FIXME: See #80255
```